apps/discovery-platform/src/constants.ts

```diff
@@ -13,6 +13,8 @@ const BASE_QUOTA = process.env.BASE_QUOTA
   ? BigInt(process.env.BASE_QUOTA)
   : BigInt(1);
 
+const SECRET = process.env.SECRET;
```

Comment:
is it supposed to work when it's undefined?

Reply:
no, it is required